README.md

```diff
@@ -81,3 +81,23 @@ this.get('scroller').scrollVertical(target, options);
 * `duration`
 * `easing`
 * `complete` -- a callback to execute once the scrolling animation is complete.
+
+## Configuration
+Some frameworks - like Google's Material Design Lite - will use a custom DOM structure to wrap the main content (e.g. for facilitating responsive design, modal overlays). For use in such environment, you can override the default scrollable element (`html, body`) with the container element that should be used by the service to set the vertical scroll position. In your router code, add the following:
+```javascript
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  i18n: Ember.inject.service(),
```

Comment:
Oh snap, my bad! Wasn't paying enough attention when copying which injection line over. Thank you! :D